.github/workflows/mlc_config.json

```diff
@@ -0,0 +1,12 @@
+{
+  "timeout": "20s",
+  "retryOn429": true,
+  "retryCount": 2,
+  "fallbackRetryDelay": "30s",
+  "aliveStatusCodes": [200, 206],
+  "ignorePatterns": [
+    {
+      "pattern": "^(https:\/\/github.com\/Ocelot-Social-Community\/Ocelot-Social\/pull\/|http:\/\/localhost)"
+    }
+  ]
+}
```

Comment:
I am against the use of this file and would like to use the disable syntax:
```
<!-- markdown-link-check-disable -->
## Section

Disable link checking in this section. Ignore this [Bad Link](https://exampleexample.cox)
<!-- markdown-link-check-enable -->
```

Reply:
Ok i was mistaken that this file resides in the main directory and therefore the connection to the workflow was not imminent visible to me. After checking again I see its within the workflow folder 👍🏻 !